day_10/task_2/BUILD

```diff
@@ -3,14 +3,17 @@ cc_binary(
     srcs = ["main.cpp"],
     deps = [
         ":pipe_maze",
-        "//path_helper"
-        # "//debug_features:debug_features"],
+        "//path_helper",
+        "//debug_features:debug_features"
```

Comment:
```suggestion
```
debug features should be removed before the merge

Reply:
done